app/views/teaching_events/show/_what-theyre-saying.html.erb

```diff
@@ -0,0 +1,17 @@
+<section class="what-theyre-saying">
+  <div class="triangle yellow">
+    <svg height="100%" width="100%" viewbox="0 0 100 100" preserveAspectRatio="none">
+      <polygon points="0,100 100,0 100,100" />
+    </svg>
+  </div>
+  <div class="subheading">
+    <h3>What they're saying</h3>
+  </div>
+  <figure>
+    <blockquote>
+      Absolutely loved this day. Very informative, inspiring and I'm
```

Comment:
Please can you replace this quote with

So useful! I got answers to questions I didn't know I had yet and I'm so inspired and excited.